QuizApp/QuizApp/UI/Quiz/QuizDifficultyLevelView.swift

```diff
@@ -0,0 +1,64 @@
+import UIKit
+
+class QuizDifficultyLevelView: UIView {
+
+    private var stackView: UIStackView!
+
+    init() {
+        super.init(frame: .zero)
+
+        createViews()
+        styleViews()
+        defineLayoutForViews()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    func set(color: UIColor, difficulty: QuizDifficultyLevel) {
+        var elements = difficulty.elements
+
+        for view in stackView.arrangedSubviews {
+            view.tintColor = elements > 0 ? color : UIColor(red: 1, green: 1, blue: 1, alpha: 0.2)
+            elements-=1
```

Comment:
`elements -= 1` Do you have SwiftLint turned on?

Reply:
Yes, it reports other warnings, but I didn't get any for this line.